example/src/screens/calendars.js

```diff
@@ -357,6 +379,26 @@ const CalendarsScreen = () => {
     );
   };
 
+  const renderCalendarWithUnderDayView = () => {
+		return (
+			<Fragment>
+				<Text style={styles.text}>Calendar with a custom view under the day component</Text>
+				<Calendar
+					style={styles.calendar}
+					current={'2020-06-16'}
+					markedDates={dayUnderViewData}
+          theme={{
+            dayContainerStyle:{  //customize style
+              //height: 32, //allow user give a height to show each week row of month as a same height.
+            },
+            //daySideLength: 38,
```

Comment:
Perhaps it would make sense to have these new example styles uncommented?

Reply:
Well, my meaning is give some data to guide user use it. but the demo data is not very pretty. also can uncommented.